webdriver-spec.html

```diff
@@ -5114,6 +5114,10 @@
       length of <var>files</var> is not equal to 1 return
       an <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
+     <li><p>Verify that each file given by the user exists. If any do
+      not, return <a>error</a> with <a>error code</a> <a>invalid
+      argument</a> detailing which files were not present.
```

Comment:
Remove “detailing which files were not present”. The `message` field is voluntary.